lib/realtime/tenants/repo/migrations/20241224161212_realtime_send_sets_config.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Realtime.Tenants.Migrations.RealtimeSendSetsConfig do
+  @moduledoc false
+  use Ecto.Migration
+
+  # We missed the schema prefix of `realtime.` in the create table partition statement
+  def change do
+    execute("""
+    CREATE OR REPLACE FUNCTION realtime.send(payload jsonb, event text, topic text, private boolean DEFAULT true ) RETURNS void
+    AS $$
+    BEGIN
+      BEGIN
+        -- Set the topic configuration
+        SET SESSION realtime.topic TO topic;
```

Comment:
Actually we do not want to set this for the whole session, only for the transaction

```
set_config('realtime.topic', 'topic', true)
```

Reply:
Or `SET LOCAL`